src/benchmarklib/benchmark_utils.hpp

```diff
@@ -110,7 +110,8 @@ struct BenchmarkConfig {
   const ChunkOffset chunk_size = 100'000;
   const EncodingConfig encoding_config = EncodingConfig{};
   const size_t max_num_query_runs = 1000;
-  const Duration max_duration = std::chrono::seconds(5);
+  const Duration max_duration = std::chrono::seconds(60);
+  const Duration warmup_duration = std::chrono::seconds(10);
```

Comment:
As of now, the default TPCH config runs every query at least once per second on nemea. Do we expect anything good from running 10s instead of 5s or even 3s?
The positives effects of running a warmup of minutes is something we don't want to measure (I'd guess) so all we want to ensure is that the query has been parsed and cached before at least once, isnt' it?